pkg/rancher-desktop/config/__tests__/settings.spec.ts

```diff
@@ -420,4 +422,239 @@ describe('settings', () => {
       expect(calculatedLockedFields).toEqual(expectedLockedFields);
     });
   });
+
+  describe('migrations', () => {
```

Comment:
This is done in `pkg/rancher-desktop/main/commandServer/__tests__/settingsValidator.spec.ts`:'should complain about unchangeable fields'

- Because a version from the future is treated like an attempt to change the fixed version field. Not perfect, and changeable, but that's where this test would reside.